### DISCUSSION TITLE
Transfer pictures to Slicer from a generic USB webcam in Linux using Plus Server

### DISCUSSION THREAD

**Discussion by lbo2021** [PlusToolkit/PlusLib#861] [Q&A]
Hi, 
I'm trying to transfer pictures to Slicer from a generic webcam **in Linux** using Plus Server and IGTlink. 

In Windows I can successfully transfer pictures to Slicer using this Plus configuration file: https://github.com/PlusToolkit/PlusLibData/blob/master/ConfigFiles/PlusDeviceSet_Server_MmfColorVideoCapture.xml

Do you know how I should do in Linux?

Thank you

**Comment by Sunderlandkyl** [comment]
You can try using the [OpenCVVideo device](http://perk-software.cs.queensu.ca/plus/doc/nightly/user/DeviceOpenCVVideo.html).
There's a sample config file for using OpenCV with a webcam here: https://github.com/PlusToolkit/PlusLibData/blob/master/ConfigFiles/PlusDeviceSet_Server_OpenCVWebcam.xml

**Reply by lbo2021** [reply]
Thank you for your fast reply.

I tried but I got the following error:

`Connect using configuration file: /home/user/PlusBuild-bin/PlusLibData/ConfigFiles/PlusDeviceSet_Server_OpenCVWebcam.xml
Server process command line: "/home/user/PlusBuild-bin/bin/PlusServer" --config-file="PlusDeviceSet_Server_OpenCVWebcam.xml" --verbose=3
|ERROR|213.684229| Server stopped unexpectedly. Return code: 1| in /home/user/PlusBuild-bin/PlusApp/PlusServerLauncher/PlusServerLauncherMainWindow.cxx(865)
Disconnect request successful
|ERROR|213.686052| Failed to start server process| in /home/user/PlusBuild-bin/PlusApp/PlusServerLauncher/PlusServerLauncherMainWindow.cxx(339)`

I tried to change the device ID from 0 to 1 but I got the same error.

Do you have any suggestion?

Thank you

**Reply by Sunderlandkyl** [reply]
Could you set the log level to Trace and upload the full PlusServer log?
